### ISSUE TITLE
branch for supporting legacy robots with firmware version 4

### ISSUE THREAD

**Issue by christian-rauch** [frankaemika/franka_ros2#49]
There are still many users of the now EOL Franka Panda robots that are stuck on firmware version 4 and libfranka 0.9.

If it is not possible to support libfranka 0.9 and >= 0.10 in parallel on the `humble` branch, then it would alternatively be very useful to have a "legacy" branch in this upstream repo to (unofficially) support Franka robots with firmware version 4 and libfranka 0.9 with backports. The tag [`v0.1.0`](https://github.com/frankaemika/franka_ros2/releases/tag/v0.1.0) is the last one that supports libfranka 0.9 as newer versions are requesting higher libfranka versions.

Could someone create a branch from this tag? My suggestions for naming would be `legacy` or `fer1` for "Franka Emika Robot 1".

**Comment by peterdavidfagan** [comment]
I think the following effort from @tingelst is aimed at addressing this need: https://github.com/tingelst/libfranka. 

I am also interested in this as our lab has the EOL Franka Panda robots that I was hoping to run with the latest ROS 2 software.

**Comment by christian-rauch** [comment]
@BarisYazici Can you please create a branch for the old "Franka Emika Robot 1"?

**Comment by christian-rauch** [comment]
@AndreasKuhner Can you create a branch for the FER (branched off from `v0.1.0`) where the community can maintain support for the old Franka robots? I don't expect Franka to provide further support, but it would be nice to have the support for the old Franka in the upstream repo.

We have a `franka_ros2` fork at https://github.com/boschresearch/franka_ros2, where we maintained support for the old Frankas with a couple of fixes.

**Comment by AndreasKuhner** [comment]
Hi @christian-rauch ,
I don't know if we want to 'officially' have a branch for that... :thinking: I will take the topic and talk to some people :)

Cheers,
Andreas

**Comment by AndreasKuhner** [comment]
Long time no answer -> so far we don't plan to have anything official. But of course feel free to share your own branches :)

**Comment by christian-rauch** [comment]
I am maintaining support for the EOL FER1 here:

https://github.com/mul-cps/franka_ros2
